ament_cmake_pytest/cmake/ament_has_pytest_cov.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Check if the Python module `pytest-cov` was found and get its version if it is.
+#
+# :param var: the output variable name
+# :type var: string
+# :param version_var: the variable name for the version of pytest-cov found, only
+#   set if it was found
+# :type version_var: string
+# :param QUIET: suppress the CMake warning if pytest-cov is not found, if not set
+#   and pytest-cov was not found a CMake warning is printed
+# :type QUIET: option
+# :param PYTHON_EXECUTABLE: absolute path to the Python interpreter to be used,
+#   default to the CMake variable with the same name returned by
+#   FindPythonInterp
+# :type PYTHON_EXECUTABLE: string
+#
+# @public
+#
+function(ament_has_pytest_cov var version_var)
```

Comment:
How about reducing the function to a single "return" variable which either contains the version number if the module was found and `FALSE` otherwise? It would probably make sense to rename the function to `ament_get_pytest_cov_version` then.

The alternative direction would be to allow `version_var` to be optional. So if the caller only cares if the module exists but doesn't need to know the version it could be called without the second variable.

Reply:
I like the first proposal. Done in aad4240ed089c0674bde9f2f8a6a9643acbbef0c.